### PR TITLE
Add a page of the types of jobs of a professional and add tests

### LINKED ISSUE #95: Add a page of the types of work of a professional

### PR DESCRIPTION
**Pull request by idosinger98**
… tests

A list of the types of jobs each professional has been created.

 Every professional can update and delete the type of job

 The changes that were made:

1.I added a type of job page

2.I added a page where you can edit job details

3.I added a page to cancel a type of job

4.Add base.html

5.Add tests
Resolve #95 

image

### CODE REVIEW

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by idoyek — APPROVED** [review]
LGTM

File: reservation/test_typeOfJob_addList.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    typeOfJob = make_typeOfJob()
+    typeOfJob.professional_id.save()
+    typeOfJob.save()
+    return typeOfJob
+
+
+@pytest.mark.django_db
+class TestTypeOfJob:
+    def test_typeOfJob_list(self, client, professional):
+        client.force_login(professional.profile_id.user_id)
```
Comment (Yarboa) [inline reservation/test_typeOfJob_addList.py:18]: same comment here, please avoid login

**Review by Yarboa — COMMENTED** [review]
Please split the form tests validity from the behavior of data flow

File: reservation/test_typeOfJob_addList.py (hunk 1 above)
Comment (idosinger98) [reply reservation/test_typeOfJob_addList.py]: Done.

**Comment by idosinger98** [comment]
> Please split the form tests validity from the behavior of data flow

Done @Yarboa

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Pull request merged**